docs/_tutorials/scenarios/scenarios.rst

```diff
@@ -310,6 +310,60 @@ supports this setting.
     # Not implemented yet
 
 
+Incremental Semantic Segmentation
+---------------------------------
+
+Brought by [Michieli et al. ICCV-W 2017](https://openaccess.thecvf.com/content_ICCVW_2019/papers/TASK-CV/Michieli_Incremental_Learning_Techniques_for_Semantic_Segmentation_ICCVW_2019_paper.pdf)
+and [Cermelli et al. CVPR 2020](https://arxiv.org/abs/2002.00718), continual learning
+for segmentation is very different from previous scenarios.
+
+Semantic segmentation aims at classifying all pixels in an image, therefore
+multiple classes can co-exist in the same image. This distinction leads to three kinds of scenarios:
+
+- **Sequential**: where for a given task T, with current classes C, the model sees all
+  images that contain at least one pixel labeled as a current classes C. If the image contains
+  future classes, yet unseen, then it is discarded. In the sequential setting, all pixels
+  are labeled, either with a old or current class label, background label (0), or
+  unknown label (255).
+- **Disjoint**: It's the same scenario as Sequential, but on one point. An image's pixel is
+  only labeled for the current classes. Therefore, during training, if an old class is
+  present in the image, its labels would be 0 (aka background). However, during the test
+  phase, all labels (current + old) are present.
+- **Overlap**: It's the same scenario as Disjoint except that the model can also
+  see images containing a future class, as long as a current class is present.
+
```

Comment:
je suis pas sur de tout comprendre dans ces 3 scenarios, ca me parait pas tres claire

Reply:
I could write more, but I guess the user is supposed to read a bit about the litterature no?

Reply:
I think the simplest, would be to add a figure to explain the different between the three settings.

Reply:
yes figures would help I think. I don't necessarily think that more should be written but it can maybe be clearer

Reply:
#103